src/Sylius/Component/Product/Model/Product.php

```diff
@@ -80,6 +80,7 @@ public function __construct()
         $this->variants = new ArrayCollection();
         $this->options = new ArrayCollection();
         $this->createdAt = new \DateTime();
+        $this->coll = new ArrayCollection();
```

Comment:
What does it mean _coll_? `Coll is an island in the Inner Hebrides, to the west of the isle of Mull.` at least my dictionary says that.